noxfile.py

```diff
@@ -265,6 +265,7 @@ def local(session):
         *test_dependencies,
         *pyproject_data["project"]["optional-dependencies"]["ssh"],
         *pyproject_data["project"]["optional-dependencies"]["nox"],
+        "-e",
```

Comment:
This shouldn't be required?

Reply:
I didn't think so either, but adding it resolved some surprising test failures locally: without it, I think `pip install ./vectors/` was performing a no-op due to the venv's installed version matching the local copy, even though the local copy had new changes since the last time the local venv was created.

Reply:
I have seen this behavior in `nox -e local` as well, although in my case it persisted even when deleting the .nox dir and a uv upgrade appeared to resolve it? Our aggressive pursuit of speed seems to have resulted in some occasionally caching issues…

Reply:
Yeah, I saw the same -- persisted even when `.nox` was blown away...